src/metamorpho/MetamorphoSnippets.sol

```diff
@@ -78,18 +77,27 @@ contract MetaMorphoSnippets {
         return withdrawQueueList;
     }
 
-    /// @notice Returns the supply cap of a market on a MetaMorpho `vault`.
+    /// @notice Returns the sum of the supply caps of markets with the same collateral `token` on a MetaMorpho `vault`.
+    /// @dev This is a way to visualize exposure to this token.
     /// @param vault The address of the MetaMorpho vault.
-    /// @param marketParams The morpho blue market.
-    function capMarket(address vault, MarketParams memory marketParams) public view returns (uint192 cap) {
-        Id id = marketParams.id();
-        cap = IMetaMorpho(vault).config(id).cap;
+    /// @param token The collateral token.
+    function totalCapAsset(address vault, address token) public view returns (uint192 totalCap) {
+        uint256 queueLength = IMetaMorpho(vault).withdrawQueueLength();
+
+        for (uint256 i; i < queueLength; ++i) {
+            Id idMarket = IMetaMorpho(vault).withdrawQueue(i);
+            MarketParams memory marketParams = morpho.idToMarketParams(idMarket);
+
+            if (marketParams.collateralToken == token) {
+                totalCap += IMetaMorpho(vault).config(idMarket).cap;
+            }
+        }
     }
 
-    /// @notice Returns the current APR (Annual Percentage Rate) of a morpho blue market.
+    /// @notice Returns the current APY of a morpho blue market.
     /// @param marketParams The morpho blue market parameters.
     /// @param market The morpho blue market state.
-    function supplyAPRMarket(MarketParams memory marketParams, Market memory market)
+    function supplyAPYMarket(MarketParams memory marketParams, Market memory market)
```

Comment:
This should also be changed in MorphoSnippets right?

Reply:
this comment was resolved twice but I don't get why ^^ This change needs to be also propagated to MorphoSnippets right?